source/documentation/gluster-hyperconverged/chap-Deploying_Hyperconverged.html.md

```diff
@@ -20,19 +20,21 @@ title: Deploying oVirt and Gluster Hyperconverged
 
 **Installing the packages on the first host**
 
-1. Subscribe to ovirt repos from http://resources.ovirt.org/pub/yum-repo/
-   For instance, to subscribe to oVirt 4.1 repo,
+1. On all 3 hosts, subscribe to ovirt repos from http://resources.ovirt.org/pub/yum-repo/
+   For instance, to subscribe to oVirt 4.1 repo.
 
         # yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm
 
-2. Install gdeploy and cockpit-ovirt that will provide a UI for the installation. gdeploy is a wrapper tool around Ansible that helps to setup gluster volumes.
+2. On all 3 hosts, install gdeploy and cockpit-ovirt that will provide a UI for the installation. gdeploy is a wrapper tool around Ansible that helps to setup gluster volumes. vdsm-gluster is a plugin to manage gluster services.
```

Comment:
Hmmm, why do we need gdeploy on all 3 hosts? also unsure about cockpit-ovirt-dashboard.

Reply:
I'm not sure either. Pinging @sabose for input! Do gdeploy and cockpit-ovirt-dashboard only have to be installed on the first host?

Reply:
gdeploy is needed only on first host. cockpit-ovirt-dashboard can be installed on all 3 hosts, as it can provide a view into the deployment from any of the hosts